src/utils/wrapInCssVar.ts

```diff
@@ -0,0 +1,3 @@
+export function wrapInCssVar(cssVariable: `--${string}`) {
```

Comment:
I don't think this abstraction is worth it, it's more boilerplate & unnecessary complexity to use than just typing `var()` where we need it